app/scripts/controllers/overview.js

```diff
@@ -540,8 +539,9 @@ angular.module('openshiftConsole')
           groupServices();
           groupPods();
           groupDeploymentConfigs();
-          groupReplicationControllers();
-          groupReplicaSets();
+          groupReplicationControllersByService();
+          groupReplicationControllersByDC();
```

Comment:
does this really need to be called here? groupRCByDC doesn't depend on services that i can see